### PR TITLE
Unify XR builds.

### PR DESCRIPTION
**Pull request by at-ninja**
Combine the `openxr` and `quest` targets into a single `xr` target.

### CODE REVIEW

File: src/android/AndroidManifest.XR.xml [hunk 1]
```diff
@@ -9,19 +9,35 @@
...
+
   <application
       android:allowBackup="true"
       android:icon="@android:drawable/ic_info"
       android:label="${appLabel}"
       android:theme="@style/Theme.AppCompat.NoActionBar"
       tools:targetApi="31">
+    <!-- Needed for Oculus devices -->
+      <uses-native-library
+          android:name="libopenxr_forwardloader.oculus.so"
```
Comment (Keenuts) [inline src/android/AndroidManifest.XR.xml:26]: What happens if we forget to include this on an oculus device? Do we have a clear error with some undefined symbol or something handled by the XR runtime, or is it crashing in a weird way?

**Review by Keenuts — COMMENTED** [review]
Fine with those changes. I like the fact that we now compile all paths for XR, as long as it doesn't require additional requirements for basic OpenXR builds (seems like no).
Just a question around the error handling in case of the missing oculus lib, but overall, I'm fine with those changes! Thanks!

**Review by apazylbe — APPROVED** [review]
Thanks! There seems to be a merge conflict, would you be able to update to latest?

**Comment by at-ninja** [comment]
> Thanks! There seems to be a merge conflict, would you be able to update to latest?

I rebased this, the merge conflict should be resolved now.

**Review by Keenuts — APPROVED** [review]
Thanks!

**Pull request merged**